test/browser/lifecycles/getDerivedStateFromError.test.js

```diff
@@ -52,8 +52,8 @@ describe('Lifecycle methods', () => {
 			}
 		}
 
-		spyAll(Receiver.prototype);
 		spyAll(Receiver);
+		spyAll(Receiver.prototype);
```

Comment:
This tests failed unless I swapped the order here. Perhaps some new class transform broke the old form? 🤷‍♂